src/node.cc

```diff
@@ -4312,8 +4321,10 @@ int Start(int argc, char** argv) {
   V8::SetEntropySource(crypto::EntropySource);
 #endif
 
+#ifndef NODE_EMBEDDED_MODE
   default_platform = v8::platform::CreateDefaultPlatform(v8_thread_pool_size);
```

Comment:
Yeah, I would rather keep CLI options, because we are throwing on unknown ones... It should be simpler for everyone!